### ISSUE TITLE
Board fails to load

### ISSUE THREAD

**Issue by alexschwantes** [NordMagnus/Trello-Folds#74]
This plugin used to work for me but now it fails to load every time. No errors are output as it essentially gets stuck in a loop waiting for the board to initialise

The Console looks like this:
```
initialize, attempt=0 tdom.js:126 
INITIALIZING NEW BOARD: t6byxXJL (old board ID: undefined) tdom.js:198 
watchForMutations, attemptCount=1 tdom.js:211 
Looking for DOM mutations during board change  tdom.js:278   
INITIALIZING NEW BOARD: t6byxXJL (old board ID: undefined) tdom.js:198 
(1680) Waiting for board to load... (newMutations=false,boardCompletelyLoaded=false) tdom.js:269 
```

For some reason it prints INITIALISING NEW BOARD twice, and then gets stuck on `Waiting for board to load...`

I've tracked down the issue to this portion of code that was commented out in a recent release.

https://github.com/NordMagnus/Trello-Folds/blob/4f9e35533f01cdaf4328fab66b46257956707967/extension/tdom.js#L232-L237

If I re-enable this line instead of defaulting `this.boardCompletelyLoaded = false`, then it loads reliably.

**Comment by sgkrasts** [comment]
Hi! I didn't see this when I posted my issue #75  Do you think mine is the same issue as yours?  I'd check to see if I get the same error, but I don't know how to do it. If you think it will help, and you can tell me how, I can check to see if the same thing is happening for me.

**Comment by alexschwantes** [comment]
@sgkrasts yes that sounds exactly like my issue. In the past I would have to refresh a few times until it worked, now it doesn't work at all and shows like yours does. 

I did install a local copy and made the change I listed above, and that does make the x show again, but if you switch boards then it fails to load and you need to refresh the page, so its far from a suitable fix for the issue.
